packages/astro/src/@types/astro.ts

```diff
@@ -1112,6 +1112,19 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		service?: ImageServiceConfig;
+			/**
+			 * @docs
+			 * @name image.service.config.limitInputPixels
+			 * @kind h4
+			 * @type {boolean}
+			 * @default `true`
+			 * @version 4.1.0
+			 * @description
+			 *
+		   * Whether or not to limit the size of images that the Sharp integration will process.
+		   *
+			 * Set `false` to bypass the default image size limit for the Sharp integration and process large images.
```

Comment:
```suggestion
			 * Set `false` to bypass the default image size limit for the Sharp image service and process large images.
```

Wish we could've linked to Sharp's docs for this option, unfortunately there's no way to make a permalink to it...